src/components/content/credential/CredentialDetails.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { Table } from 'antd';
+import React from 'react';
+import { CredentialVariable } from '../../../xpanse-api/generated';
+import { ColumnsType } from 'antd/es/table';
+
+function CredentialDetails({ credentialDetails }: { credentialDetails: CredentialVariable[] }): JSX.Element {
+    const columns: ColumnsType<CredentialVariable> = [
```

Comment:
table is not fitting the modal size

![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/61e9c203-12ec-41b8-a150-37d629f035b0)